src/libYARP_dev/src/devices/AnalogWrapper/AnalogWrapper.h

```diff
@@ -94,17 +100,28 @@ namespace yarp{
  * frame_id      r_shoulder
  * \endcode
  *
- * Configuration file using .xml format.
+ * Configuration file using .xml format(skin example).
  *
- * \code{.unparsed}
+ * \code{.xml}
+ *	<device name="left_arm_skin_wrapper" type="skinWrapper">
+ *		<param name="period">       20  				</param>
+ *		<param name="total_taxels"> 1344				</param>
+ *
+ *		<paramlist name="ports">
+ *		  <elem name="left_hand"> 	0   191  0 191</elem>
+ *		  <elem name="left_forearm"> 	192 575  0 383</elem>
+ *		  <elem name="left_arm"> 	576 1343 0 767</elem>
+ *		</paramlist>
+ *
+ *		<action phase="startup" level="5" type="attach">
+ *		    <paramlist name="networks">
+ *		<!-- The param value must match the device name in the corresponding emsX file -->
+ *		        <elem name="FirstSetOfSkins">  left_arm_skin </elem>
+ *		    </paramlist>
+ *		</action>
  *
- *  * <group name="ROS">
- *     <param name="useROS">         true                           </param>    // use 'only' if you want only ROS topic and NOT yarp ports
- *     <param name="ROS_topicName">  /left_arm/forceTorque          </param>
- *     <param name="ROS_nodeName">   /torquePublisher               </param>
- *     <param name="ROS_msgType">    geometry_msgs/WrenchedStamped  </param>
- *     <param name="frame_id">       r_shoulder                     </param>
- * </group>
```

Comment:
Please do not remove the ROS xml example, it is still valid.
Just add a separated  `code` section in the doxy